pkg/shell/index.html

```diff
@@ -94,52 +94,6 @@
             </div>
         </div>
 
-        <div class="modal" id="edit-host-dialog" tabindex="-1" role="dialog" data-backdrop="static">
-            <div class="modal-dialog">
-                <div class="modal-content">
-                    <div class="modal-header">
-                        <h4 class="modal-title" translate>Edit host</h4>
-                    </div>
-                    <div class="modal-body">
-                        <div>
-                            <table class="form-table-ct">
-                                <tr>
-                                    <td class="top" translate="yes">Host name</td>
-                                    <td>
-                                        <input class="form-control" id="edit-host-name"/>
-                                    </td>
-                                    <td></td>
-                                </tr>
-                                <tr id="edit-host-user-row">
-                                    <td class="top" translate="yes">User</td>
-                                    <td>
-                                        <input class="form-control" id="edit-host-user"/>
-                                    </td>
-                                    <td>
-                                        <a tabindex="0" role="button" data-toggle="popover"
-                                            data-trigger="focus" data-placement="bottom" translate="data-content"
-                                            data-content="Leave blank to connect to this machine as the currently logged in user. If you enter a different username, that user will always be used when connecting to this machine.">
```

Comment:
This information seems to be missing. Should be present in both add new and edit modals.
Also see https://github.com/cockpit-project/cockpit/issues/14734

Reply:
Yes, good point, let's add it back.

Reply:
In a separate PR. :-)

Reply:
ok :) We have issue opened anyway :)

Reply:
#14858